docs/source/training.mdx

```diff
@@ -347,6 +348,9 @@ Just like how you need to add an evaluation function to [`Trainer`], you need to
 
 ```py
 >>> metric = load_metric("accuracy")
+
+>>> progress_bar = tqdm(range(num_eval_steps))
```

Comment:
There is no need to define the progress bar separately here, you can just wrap the `eval_dataloader` below in a `tqdm`.